fabric-object-builders-v0/src/main/java/net/fabricmc/fabric/api/block/FabricBlockSettings.java

```diff
@@ -68,16 +68,27 @@ public static FabricBlockSettings copyOf(Block.Settings settings) {
 
 	/* FABRIC HELPERS */
 
+	/**
+	 * Makes the block breakable by any tool if {@code breakByHand} is set to true.
+	 */
```

Comment:
I get that docs are nice, but the v0 block settings should be deprecated, so unsure if the effort is needed for it

Reply:
Just a few neat javadocs, can't go wrong with that tbh